SPECS/rsyslog/rsyslog.logrotate

```diff
@@ -0,0 +1,40 @@
+/var/log/syslog
+{
+    rotate 7
+    daily
+    missingok
+    notifempty
+    delaycompress
+    compress
+    postrotate
+        /usr/bin/systemctl kill -s HUP rsyslog.service >/dev/null 2>&1 || true
+    endscript
+}
+
+ 
+
+/var/log/mail.info
+/var/log/mail.warn
+/var/log/mail.err
+/var/log/mail.log
+/var/log/daemon.log
+/var/log/kern.log
+/var/log/auth.log
+/var/log/user.log
+/var/log/lpr.log
+/var/log/cron.log
+/var/log/debug
+/var/log/messages
```

Comment:
I think our `rsyslog.conf` config doesn't align with this logrotate setup. We define a few different logs such as `/var/log/secure` , `/var/log/maillog` `/var/log/spooler` which I don't believe will match any of these.

Reply:
Thanks @dmcilvaney for pointing out the mismatch! logrotate setup is modified to align with the definitions in rsyslog.conf.  logrotate setup for `/var/log/syslog` and `/var/log/rsyslog-warn` has been removed as they are not defined in configuration and doubled checked in live VMs these files don't exist. Please take another look.